tests/base.py

```diff
@@ -159,8 +159,8 @@ def setUp_sqlite_resolver_realm(self, sqlite_file, realm):
         parameters = {'resolver': "sqlite_resolver",
                       "type": "sqlresolver",
                       'Driver': 'sqlite',
-                      'Server': '/tests/testdata/',
-                      'Database': sqlite_file,
+                      'Server': '',
+                      'Database': 'tests/testdata/' + sqlite_file,
```

Comment:
Did not test it right now but are be backward compatible?

Reply:
With older configurations. So does it break existing configs that use the separated definitions?

Reply:
Ah ok yes it would break if someone defined an SQLResolver with an SQLite DB like this. I assumed no one outside of the tests would do something like this because it's not documented anywhere and the WebUI has clear definitions for the fields. Is there an update mechanism which i could use to parse it and convert it if needed ? Like the db upgrade functionality. Otherwise i would try to add logic to the _create_connection_url method to detect and correct this.

Reply:
Maybe a SQL Migration could work. Let me discuss it on Friday with the remaining team.

Reply:
We are quite unsure if this config is really not used by anyone. Would prefer to have an migration or at lease some sort of backward compatibility.